src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationRefresher.cs

```diff
@@ -47,6 +47,7 @@ public interface IConfigurationRefresher
         /// A random delay is added before the cached value is marked as dirty to reduce potential throttling in case multiple instances refresh at the same time.
         /// </summary>
         /// <param name="maxDelay">Maximum delay before the cached value is marked as dirty. Default value is 30 seconds.</param>
```

Comment:
Do we have a document discussing how to migrate from SetDirty to ProcessPushNotification?

Reply:
We will update the public doc to use ProcessPushNotification API instead of SetDirty ([Matthew's PR](https://github.com/MicrosoftDocs/azure-docs/pull/86403)).  Do you think we need additional migration documentation somewhere?

Reply:
I don't think so. Pointing to that in the release notes should be okay.

Reply:
Just wanted to make sure details on how to use the new API exist so there's not unnecessary friction when moving to non-obsolete API.

Reply:
Sounds good, thanks!